metadata-ingestion/src/datahub/utilities/sqlglot_lineage.py

```diff
@@ -825,3 +825,43 @@ def sqlglot_lineage(
                 table_error=e,
             ),
         )
+
+
+def create_lineage_sql_parsed_result(
```

Comment:
please also refactor tableau to use this method

Reply:
done